service/storage_proxy.cc

```diff
@@ -340,7 +340,7 @@ class cas_mutation : public mutation_holder {
     virtual future<> apply_locally(storage_proxy& sp, storage_proxy::clock_type::time_point timeout,
             tracing::trace_state_ptr tr_state) override {
         tracing::trace(tr_state, "Executing a learn locally");
-        return paxos::paxos_state::learn(_schema, *_proposal, timeout, tr_state);
+        return paxos::paxos_state::learn(get_local_storage_proxy(), _schema, *_proposal, timeout, tr_state);
```

Comment:
You have `sp` here. Why call `get_local_storage_proxy`?

Reply:
Good catch, I missed this in my review. @avikivity I already committed this patch, so you can fix it as a followup.